hosted_control_planes/hcp-authentication-authorization.adoc

```diff
@@ -9,9 +9,34 @@ toc::[]
 The {product-title} control plane includes a built-in OAuth server. You can obtain OAuth access tokens to authenticate to the {product-title} API. After you create your hosted cluster, you can configure OAuth by specifying an identity provider.
 
 include::modules/hcp-configuring-oauth.adoc[leveloffset=+1]
+
 include::modules/hcp-configuring-oauth-console.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * To know more about supported identity providers, see xref:../authentication/understanding-identity-provider.adoc#understanding-identity-provider["Understanding identity provider configuration"] in _Authentication and authorization_.
+
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+3]
```

Comment:
Might consider bumping this to a +2? Right now it looks like it's part of "Configuring the OAuth server", but IIUC it's not really part of that
```suggestion
include::modules/cco-ccoctl-configuring.adoc[leveloffset=+2]
```

Alternatively, you could make a subheading at the `==` level to put all the CCO stuff inside to more closely match the structure of these tasks in the install books. 

So, something like:
* Configuring the OAuth server for a hosted cluster by using the CLI
* Configuring the OAuth server for a hosted cluster by using the web console
* Configuring an {hcp} cluster to use the AWS Security Token Service
  * Configuring the Cloud Credential Operator utility
  * Creating AWS resources with the Cloud Credential Operator utility
    * Creating AWS resources with a single command
    * Creating AWS resources individually
  * Incorporating the Cloud Credential Operator utility manifests

This was a bit harder to show with a PR review diff, but I sketched it out [in my editor here](https://github.com/openshift/openshift-docs/compare/main...jeana-redhat:openshift-docs:cco-hcp?expand=1) if you want to see what I'm thinking :slightly_smiling_face: 

Local render TOC looks like this:
![image](https://github.com/openshift/openshift-docs/assets/61474374/7904efe6-933e-432d-95a4-7f4ac5b57b8a)